admin/server/bookmarks.go

```diff
@@ -96,16 +96,22 @@ func (s *Server) CreateBookmark(ctx context.Context, req *adminv1.CreateBookmark
 		permissions.ReadProd = true
 	}
 
+	if !permissions.ManageProject {
+		req.IsGlobal = false // only users that can manage the project can
+	}
```

Comment:
Better to return an error when permissions are breached than have (maybe unexpected) side effects.

Like, if a reader role sets `is_global=true`, it probably implies a missing check on the frontend